### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stmcpeters**
- updated README with installation instructions, technologies used, features, contributions. 
- added placeholder comment for stretch goals

are there any new features or changes you'd like to add to this project in the future so i can make note in the stretch goals section? :)

### CODE REVIEW

**Review by ChasVanDav — APPROVED** [review]
Steph this is amazing and so organized and well-written! Thank you for improving the README for the Unicorn Weather App!! Coffee's on me when we meet. :)

File: unicorn-weather-app/README.md [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
+- navigate to `cd client`
+- install dependencies `npm install`
+6. Start the React development server using `npm run dev`
+7. Sign up/log in to get your own API key from [OpenWeatherAPI](https://openweathermap.org/api)
+### API Endpoints
+- GET /
+  - Fetches weather data for a specified city.
+  - Query parameter: city (optional, defaults to "Honolulu").
+### Stretch Goals
+--touch base about any stretch goals--
```
Comment (stmcpeters) [inline unicorn-weather-app/README.md:48]: add possible stretch goals/new features to add in the future
- media queries for mobile devices 
- hiding sensitive info in .env file

**Review by stmcpeters — COMMENTED** [review]
i added some stretch goals we can add onto your README :)

File: unicorn-weather-app/README.md (hunk 1 above)
Comment (ChasVanDav) [reply unicorn-weather-app/README.md:48]: These are great stretch goals that I will be implementing when time allows! Thank you!

**Pull request merged**